package.json

```
@@ -27,13 +27,16 @@
     "John Otander <johnotander@gmail.com> (https://johno.com)"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
   ],
+  "types": "types/index.d.ts",
   "dependencies": {
-    "mdast-util-to-hast": "^8.0.0"
+    "mdast-util-to-hast": "^9.0.1"
```

```suggestion
    "mdast-util-to-hast": "^9.1.0"
```

This would fix the CI failure, right?

---

Yep, thanks!